VowelsAndDuplicates.js

```diff
@@ -1,4 +1,16 @@
 function duplicatesAndVowels(name) {
-  
+  //sorting out vowels in a string
+  const vowels = name.match(/[aeiouAEIOU]/gi);
+  //   removing duplicate vowels from the string
+  const uniqueVowels = [...new Set(vowels)].join("");
+  //   getting duplicates from the original string
+  const numberOfDuplicates = name

```

Comment:
Good job on this. 
But we are splitting lucky enough using an empty delimiter which is O(StringLength), then sorting, then joining and matching. These can be expensive operations if we have very long strings, Can you think of a way you can count duplicates in a string without doing all those operations?

Reply:
working on it.

Reply:
Great